src/bp/lang-server/api.ts

```diff
@@ -175,69 +177,72 @@ export default async function(options: APIOptions) {
         throw new BadRequestError('Param `tokens` is mandatory (must be an array of strings)')
       }
 
-      const result = await options.service.vectorize(tokens, lang)
+      const result = await options.languageService.vectorize(tokens, lang)
       res.json({ language: lang, vectors: result })
     } catch (err) {
       next(err)
     }
   })
 
   const router = express.Router({ mergeParams: true })
-
-  router.get('/list', (req, res, next) => {
-    const items = options.manager.inProgress.map(x => ({
-      status: x.getStatus(),
+  router.get('/', (req, res, next) => {
+    const downloading = options.downloadManager.inProgress.map(x => ({
       lang: x.lang,
-      id: x.id,
-      downloadedSize: x.downloadedSize,
-      fileSize: x.fileSize
+      progress: {
+        status: x.getStatus(),
+        downloadId: x.id,
+        size: x.downloadSizeProgress
```

Comment:
Is it a percentage or the actual downloaded size ?

Reply:
actual download size